homeassistant/components/bmp280/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "bmp280",
+  "name": "Bosch BMP280 Environmental Sensor",
+  "documentation": "https://www.home-assistant.io/components/bmp280",
+  "dependencies": [],
+  "codeowners": ["@belidzs"],
+  "requirements": ["adafruit-circuitpython-bmp280==3.1.1", "RPi.GPIO==0.7.0"],
```

Comment:
Why does this integration needs `RPi.GPIO`?

Reply:
`adafruit-circuitpython-bmp280` [depends](https://github.com/adafruit/Adafruit_CircuitPython_BMP280/blob/master/requirements.txt) on `Adafruit-Blinka` while it [conditionally depends](https://github.com/adafruit/Adafruit_Blinka/blob/master/requirements.txt) on `RPi.GPIO` (on arm architecture anyway)

I wasn't sure if these conditional requirements will be automatically resolved by home-assistant but a tutorial written by the library author explicitly mentioned it as a requirement so I added it to the requirements as well.

As it is [commented out](https://github.com/home-assistant/home-assistant/blob/399173e3b3920a035fb36b7c5fc8933b6ee51ffe/script/gen_requirements_all.py#L41) by `gen_requirements_all.py` anyways I think it can be safely removed.

Reply:
@fabaff is this blocking the merge?